pkgs/development/python-modules/advocate/default.nix

```diff
@@ -0,0 +1,42 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, ndg-httpsclient
+, netifaces
+, pyasn1
+, pyopenssl
+, requests
+, six
+, urllib3
+}:
+
+buildPythonPackage rec {
+  version = "1.0.0";
+  pname = "advocate";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-G/EXDkEzQnmZZYAynFlOAXVAqw6vehUjI+dD8KhaNT0=";
+  };
+
```

Comment:
Can we add an pythonImportsCheck ad least?

Reply:
Done